src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/commands/mobile/unsubscribe_vehicle_request_test.cc

```diff
@@ -39,6 +39,8 @@
 #include "application_manager/mock_application_manager.h"
 #include "application_manager/mock_message_helper.h"
 #include "mobile/unsubscribe_vehicle_data_request.h"
+#include "resumption/last_state_wrapper.h"
+#include "resumption/last_state_wrapper_impl.h"
```

Comment:
@YarikMamykin WHy do we need both headers above?

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/b1dca8ac899a638cf8878c402edcead1a66c1252